github/Repository.pyi

```diff
@@ -295,6 +308,14 @@ class Repository(CompletableGithubObject):
     def get_contributors(
         self, anon: Union[str, _NotSetType] = ...
     ) -> PaginatedList[NamedUser]: ...
+    def get_deployment(self, id_: int) -> Any: ...
```

Comment:
I mean, surely this is Deployment, not Any. However, I don't want to hold this PR any further.

Reply:
Good catch. Fixed in  https://github.com/PyGithub/PyGithub/pull/1767/